src/server/database/models/index.js

```diff
@@ -28,9 +28,9 @@ dates.belongsTo(workers, {
 workers.hasMany(dates, { foreignKey: 'worker_name' });
 
 participantCourses.belongsTo(courses, {
-  onDelete: 'CASCADE', foreignKey: 'course_id', targetKey: 'id',
+  onDelete: 'CASCADE', foreignKey: 'course_name', targetKey: 'course_name',
 });
-courses.hasMany(participantCourses, { foreignKey: 'course_id' });
+courses.hasMany(participantCourses, { foreignKey: 'course_name' });
```

Comment:
I think `course_id` might be a safer foreignKey than `course_name` for this relationship, because even though course_name is now unique, I could see situations where a given course would have its name updated (for example, an admin decides to change a course's name from "Level 1 Certificate in Construction Skills (Painting & Decorating)" to "Level 1 Certificate in Construction Skills (Painting and Decorating)".  When that happens, if course_name is the foreign_key, all of the existing relationships in the db, from courses -> participantCourses for that old name, would be broken (because they'd be pointing at the old name, essentially).
What do you think? Is it possible to change back to course_id?

Reply:
Nevermind, I was wrong, I was thinking of targetKey, not foreignKey 🤦‍♂️   Sorry!

Reply:
This is can simplify our work :smile: 
and I was make a test to make sure that it can update `course_name` in the participantCourse table if I was making any change in the original name from the courses table and it was happened :ok_hand:

Reply:
I'm happy to help.  You're all doing great work!

Reply:
We are very fortunate for receiving your feedback